write-scm-version.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# write-scm-version.sh exists because `proxy.py``
+# auto-detects it's next version from git.  Hence,
+# for `proxy.py` to work `proxy/common/_scm_version.py`
+# file must be auto-generated with necessary information.
+#
+# For CI/CD, this file is generated via `tox` integration.
+# For local development (without editable install), you
```

Comment:
I'd argue, that for local development, an editable install must be used which would take care of this w/o the need for this script.

Reply:
Agreed.  I'll most likely remove this bootstrapping once readme is updated with editable install as go-to developer setup step.